src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -240,7 +240,7 @@ private static void validateTableAccess(Client engineClient, JSONObject mapping,
                     hasAccess = true;
                 }
             } catch (DataServiceException e) {
-                databaseTableErrorList.add(String.format("Database:%s Table:%s | table not found", database, table));
+                databaseTableErrorList.add(String.format("Caught exception while trying to validate access to Database '%s' Table '%s', with message '%s'", database, table, e.getMessage()));
```

Comment:
we should add here the full exception as string - not just the message - in the current SDK version message is just "Error in post request" - we might want to just pop the version .